src/bb-functions-publish.ts

```diff
@@ -205,10 +204,9 @@ const publishFunctions = async (
     {},
   );
 
-  let promises: Promise<void | string | object | null>[] = Object.keys(
-    metaData,
-  ).map(
-    (name: string): Promise<string | object | null> => {
+  await Object.keys(metaData).reduce(
```

Comment:
I would use something like this:

`Promise.all(Object.keys(metaData).map(m => new Promise(.....))`

Reply:
Thanks for the tip. I have changed it accordingly.